todo-app/src/component/header.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+class Header extends Component {
```

Comment:
In some cases you're using functional components, and here you're using a class-based component, even though no state or class properties are used. I understand you may just be experimenting with different ideas, but when you eventually write a "real" codebase, it's a good idea to make a rule for yourself and stick to it.